datalab/mlalpha/_package_runner.py

```diff
@@ -0,0 +1,84 @@
+# Copyright 2017 Google Inc. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License. You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software distributed under the License
+# is distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express
+# or implied. See the License for the specific language governing permissions and limitations under
+# the License.
+
+"""Implements running Datalab ML Solution Packages."""
+
+import inspect
+import google.cloud.ml as ml
+import os
+import shutil
+import subprocess
+import sys
+import tempfile
+
+
+PACKAGE_NAMESPACE = 'datalab_solutions.'
```

Comment:
Little odd the name ends in '.'

It looks like an error later when you don't do PACKAGE_NAMESPACE + '.' + self....